src/main/java/gmail/mishchuk/lesson40_FI/LogicToFI.java

```diff
@@ -0,0 +1,56 @@
+package gmail.mishchuk.lesson40_FI;
+
+import java.util.Random;
+
+public class LogicToFI {
+
+//    public static boolean getPrimeNumberElvisStyle (int num) {
+//        if (num <= 1 ? false);
+//        for (int i = 2; i * i <= num; i++) {
+//            if (num % i == 0 ? false : true);
+//            }
+//        return true;
+//    }
+
+    public static boolean getPrimeNumber(int numberToCheck) {
+        if (numberToCheck <= 1) {
+            return false;
+        }
+        for (int i = 2; i * i <= numberToCheck; i++) {
+            if (numberToCheck % i == 0) {
+                return false;
+            }
+        }
```

Comment:
Можна ще зробити так

```suggestion
        for (int i = 2; i <= Math.sqrt(numberToCheck); i++) {
            if (numberToCheck % i == 0) {
                return false;
            }
        }
```

Reply:
я таке знайшов був на просторах інтернету, але воно здалось якимсь "цу компліцірт"))) тому пішов від зворотнього